opslevel/resource_opslevel_check_base.go

```diff
@@ -30,6 +30,32 @@ type CheckCodeBaseResourceModel struct {
 	Owner       types.String `tfsdk:"owner"`
 }
 
+func NewCheckCodeBaseResourceModel(check opslevel.Check, planModel CheckCodeBaseResourceModel) CheckCodeBaseResourceModel {
```

Comment:
99% sure its not ALWAYS the planModel passed in here

```suggestion
func NewCheckCodeBaseResourceModel(check opslevel.Check, givenModel CheckCodeBaseResourceModel) CheckCodeBaseResourceModel {
```